src/Symfony/Bundle/SecurityBundle/SecurityBundle.php

```diff
@@ -38,7 +37,6 @@
 use Symfony\Bundle\SecurityBundle\DependencyInjection\Security\Factory\X509Factory;
 use Symfony\Bundle\SecurityBundle\DependencyInjection\Security\UserProvider\InMemoryFactory;
 use Symfony\Bundle\SecurityBundle\DependencyInjection\Security\UserProvider\LdapFactory;
-use Symfony\Bundle\SecurityBundle\DependencyInjection\SecurityExtension;
```

Comment:
shouldn't this be kept for alpha ordering?

Reply:
Afaik, these changes are done by fabbot's php cs fixer (I never reorganize use statements myself).
looking at the diff, I think PHP cs fixer now ingores the `\` characters when determining the alpha order.

Reply:
looking at the other alpha reordering, that might be due to the mysteries of conflict resolutions during a rebase
please revert (and the others too)